docs/getting-started-on-snowplow-open-source/telemetry/index.md

```diff
@@ -50,6 +50,7 @@ At the moment, opt-out telemetry is present in the following:
 * Enrich ([Enrich Kinesis](/docs/pipeline-components-and-applications/enrichment-components/enrich-kinesis/index.md), [Enrich PubSub](/docs/pipeline-components-and-applications/enrichment-components/enrich-pubsub/index.md), [Enrich Kafka](/docs/pipeline-components-and-applications/enrichment-components/enrich-kafka/index.md), [Enrich RabbitMQ](/docs/pipeline-components-and-applications/enrichment-components/enrich-rabbitmq/index.md)).
 * RDB Loader ([Transofmer Kinesis](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/stream-transformer/transformer-kinesis/index.md), [Transformer PubSub](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/transforming-enriched-data/stream-transformer/transformer-pubsub/index.md), [Redshift Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/redshift-loader/index.md), [Snowflake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/snowflake-loader/index.md), [Databricks Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/databricks-loader/index.md)).
 * Snowplow Mini for [AWS](/docs/pipeline-components-and-applications/snowplow-mini/setup-guide-for-aws/index.md) and [GCP](/docs/pipeline-components-and-applications/snowplow-mini/setup-guide-for-gcp/index.md).
+* [Snowbridge](/docs/pipeline-components-and-applications/snowbridge/index.md)
```

Comment:
1. It bothers me that there is no period at the end (all other list items have it)
2. I am following the convention of placing the telemetry notice not in the homepage of the component (which can be easily overlooked), but instead in the page that shows how to run it. Could you do the same?

Reply:
In this case we have `getting-started/quickstart`, and `configuration/index` - which page would you put it on?

Reply:
I would put the notice in both quickstart and “distribution and deployment”, with this page linking to the latter. Configuration already describes the telemetry settings, so there is no need for a notice there.

Reply:
👍 will do, thanks

Reply:
Done. :)